OutOfSchool/OutOfSchool.WebApi.Tests/Validators/CustomUkrainianNameAttributeTests.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+using OutOfSchool.Common.Validators;
+
+namespace OutOfSchool.WebApi.Tests.Validators;
+
+[TestFixture]
+public class CustomUkrainianNameAttributeTests
+{
+    public static IEnumerable<string> ValidCommonSingleNamesStrings =>
+    [
+        "Анастасія", "Марія", "Софія", "Вероніка", "Вікторія",
+        "Соломія", "Ангеліна", "Злата", "Ганна", "Єва", "Варвара",
+        "Поліна", "Аліса", "Олександра", "Мілана", "Дарина", "Аріна",
+        "Марина", "Діана", "Катерина", "Артем", "Олександр", "Максим",
+        "Дмитро", "Матвій", "Назар", "Богдан", "Марк", "Владислав",
+        "Михайло", "Володимир", "Тимофій", "Іван", "Давид", "Андрій",
+        "Олексій", "Макс", "Денис", "Антон", "Вадим",
+        "Ян", "Як", "Мар'яна", "Валер'ян", "Мар'ян",
+        "О'Коннор", "Оставф'єнко", "Прокоп'єнко",
+        "Васильович", "Петрович", "Владиславович", "Михайлович",
+        "Валер'янович", "Мар'янович", "Прізвище'запострофом",
+        "Дар'я", "Ємельян", "Ґар'єнко", "Ібраґімович", "Ївашко", "Приїжко",
+    ];
+
+    public static IEnumerable<string> NonUkrainianSingleNameStrings =>
+    [
+        "Валер'ian", "Veler'ян",
+        "Anastasia", "Mariia", "Vladyslav", "Dmytro",
+        "明美", "明里", "愛子", "金锦津錦", "金澤", "김연아",
+        "Рыбаков", "Рыбак", "Объектов", "Объект", "Ёженов", "Проёжов", "Эвклид", "Проэкт",
+    ];
+
+    public static IEnumerable<object[]> ValidComminSingleNamesTestParams =>
```

Comment:
(Mistype) `ValidComminSingleNamesTestParams` probably should be `...Common...` not `...Commin...`